### ISSUE TITLE
False positive: [EA] Written value is same as default one, consider removing this assignment.

### ISSUE THREAD

**Issue by MisatoTremor** [kalessil/phpinspectionsea#1549]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) 4.0.3 |
| Language level | PHP 7.4                        |

Current behaviour
---
The code
```php
class A {
    private ?string $foo;
    private ?A $bar;
    private ?DateTime $baz;

    public function __construct()
    {
        $this->foo = null;
        $this->bar = null;
        $this->baz = null;
    }
}
```
still yields "[EA] Written value is same as default one, consider removing this assignment." for all 3 properies.

Expected behaviour
---
No warning

Environment details
---
PHPStorm 2020.1 (201.6668.153)

**Comment by kalessil** [comment]
Thank you @MisatoTremor.

**Comment by kalessil** [comment]
Fixed!
